src/app.ts

```diff
@@ -0,0 +1,295 @@
+import assert from 'assert';
```

Comment:
Left a few comments but by-and-large this file looks like normal, quality Node code. Nice work!